src/Zio/FileSystems/MemoryFileSystem.cs

```diff
@@ -1744,7 +1744,8 @@ public FileNode(MemoryFileSystem fileSystem, DirectoryNode parentNode, string? n
             }
             else
             {
-                Attributes = FileAttributes.Archive;
+                // Mimic OS-specific attributes.
+                Attributes = PhysicalFileSystem.IsOnWindows ? FileAttributes.Archive : FileAttributes.Normal;
```

Comment:
Breaking change: Linux doesn't have the `FileAttributes.Archive` attribute, so I modified the MemoryFileSystem to mimic the OS filesystem. Is this OK?